guides/common/modules/ref_ipv6-and-ipv4-requirements.adoc

```diff
@@ -0,0 +1,43 @@
+[id="ipv6-and-ipv4-requirements"]
+= IPv6 and IPv4 requirements
+
+You can install {Project} in an IPv4 network or in an IPv6 network.
+ifdef::satellite[]
+Dual-stack {Project} installation that uses both IPv4 and IPv6 is not supported.
+endif::[]
+
+The following requirements apply to installations in an IPv4 network:
+
+* Ensure an IPv6 loopback is configured on the base system.
+The loopback is typically configured by default.
+Do not disable it.
+* Do not disable IPv6 in kernel by adding the `ipv6.disable=1` kernel parameter.
+
+The following requirements apply to installations in an IPv6 network:
+
+ifdef::satellite[]
+* Deploy an external HTTP proxy server that supports both IPv4 and IPv6.
+This is required because Red Hat Content Delivery Network distributes content only over IPv4 networks, therefore you must use this HTTP proxy to pull content into the {Project} on your IPv6 network.
+You must configure {Project} to use this dual-stack (supporting both IPv4 and IPv6) HTTP proxy as the default HTTP proxy.
+endif::[]
+ifndef::foreman-deb,foreman-el,satellite[]
+* If you rely on content from IPv4-only networks, use an external dual-stack HTTP proxy.
+Configure {Project} to use this dual-stack (supporting both IPv4 and IPv6) HTTP proxy as the default HTTP proxy.
+endif::[]
+* {Project} does not support configuring an HTTP proxy using a direct IPv6 address.
+Instead, configure the HTTP proxy with a FQDN that resolves to the IPv6 address.
+Using an IPv6 address as the HTTP proxy URL causes it to fail.
+
+If you also intend to provision hosts in an IPv6 network, consider the following requirements:
+
+* Deploy an external DHCPv6 server and configure it manually to communicate with the network boot process and to manage IP address assignment because {Project} cannot integrate with a DHCPv6 server and manage its configuration.
+For more information about DHCPv6 server configuration, see {ProvisioningDocURL}options-in-unmanaged-dhcpv6[Options in unmanaged DHCPv6] in _{ProvisioningDocTitle}_.
+* Although {Project} provisioning templates include IPv6 support for PXE and HTTP (iPXE) provisioning, the only tested and certified provisioning workflow is the UEFI HTTP Boot provisioning.
```

Comment:
I suggest that you reword this from "although X, so Y" to "You use UEFI HTTP Boot provisioning." to make it sound more like a requirement. Feel free to add the remaining information into an addional line. Right now, this sounds more like a suggestion to me than a hard technical requirement.

Reply:
Actually, I think we can merge this with the bullet point below.